### ISSUE TITLE
Accordion component: + to expand doesn't work (with anchorlink?)

### ISSUE THREAD

**Issue by meols** [equinor/energyvision#858]

**Comment by wenche** [comment]
@SvSven  Can you test this and see if it works?

----
We have changed the code to the following
- An end user clicks on an internal link with a hash tag
- The end user interacts with an accordion (or the search field/tab) and we change the url -> the hash is removed because it's no longer relevant

I don't know yet if this covers all the edge cases. We will keep an eye on this as we continue with the search :)

**Comment by adneforslund** [comment]
Also seems like the whole thing reloads if one goes to https://energyvision.app.radix.equinor.com/careers/faq and clicks on a +

**Comment by SvSven** [comment]
Can confirm that going to the URL @meols provided earlier and clicking a random accordion now works as expected.
It removes the anchor from the URL, opens the accordion and adds the accordion's parameters to the URL.

**Comment by wenche** [comment]
@meols Can you verify this?

**Comment by meols** [comment]
@wenche - I can confirm that accordions with an anchor link expands and now are working. But it seems a bit strange that the url is changing from the anchor name to "something else" after expanding the accordion, and  just a "?" being added to the url when closing the accordion. Also, if I go to the page via an anchor link (e.g. https://energyvision.app.radix.equinor.com/about-us/corporate-executive-committee-cvs#paal-eitrheim-cv) and don't expand the accordion, but scroll further up or down the page, and then click enter again after the anchor that's still there in the url, I don't go to the anchor point.
We also see the last issue - that scrolling up or down and then enter after an anchor link in the url doesn't take you to the anchor point immediately. But after pressing enter three/four times you suddenly go to the anchor point. (https://energyvision.app.radix.equinor.com/sustainability/impact-assessments#haltenpipe)

**Comment by SvSven** [comment]
@meols @wenche 

The URL is changing because the accordion has functionality that allows you to set the open/closed state when linking to a page. For example in this case: https://energyvision.app.radix.equinor.com/about-us/corporate-executive-committee-cvs?38d56e74f0d7=0 the first accordion is set to open, while by default if you land on that page the accordion would be closed.

Unfortunately this behaviour conflicts with the anchor link - we can probably get them both working in harmony, however that would potentially take quite some time and rewriting, so we decided to remove the anchor from the URL in order to have the accordions work properly.

With regards to the second point: not sure if I fully understand. I'm not able to reproduce this either.
Here is what I tried:
1. Navigate to https://energyvision.app.radix.equinor.com/sustainability/impact-assessments#haltenpipe
2. Scroll down the page
3. Don't interact with any accordion
4. Press enter (nothing happens)
5. Repeatedly press enter (nothing happens)

And that is the expected behaviour? Unless there is some browser extension that reacts on the enter key or if you are filling out a form, have a button selected with the tab key, or something similar then the enter key shouldn't really do anything. But maybe we're misunderstanding.
